### ISSUE TITLE
Entity framework produces libSystem.native error

### ISSUE THREAD

**Issue by anthonystanton** [dotnet/efcore#27669]

**Comment by sfratt** [comment]
Same issue for me. I also am on macOS 10.13 and recently updated to .NET SDK 5.0.406.

**Comment by anthonystanton** [comment]
Tried uninstalling latest .NET SDK and Runtime however still get the error.

@sfratt Have you had any luck with any solutions?

**Comment by ajcvickers** [comment]
This issue is lacking enough information for us to be able to fully understand what is happening. Please attach a small, runnable project or post a small, runnable code listing that reproduces what you are seeing so that we can investigate.

**Comment by iezed** [comment]
Hi, it also happens by launching dotnet ef, without project, on macOS 10.13.

**Comment by anthonystanton** [comment]
> 

As @iezed has said, simply launching dotnet ef produces this error. 

A brand new project with no added code will produce the same error when launching dotnet ef.

**Comment by anthonystanton** [comment]
> This issue is lacking enough information for us to be able to fully understand what is happening. Please attach a small, runnable project or post a small, runnable code listing that reproduces what you are seeing so that we can investigate.

As @iezed has said, simply launching dotnet ef produces this error.

A brand new project with no added code will produce the same error when launching dotnet ef.

**Comment by iezed** [comment]
The bug appeared with the recent version of net SDK 3.1.417 on mac 10.13, it's not relate at donet 5 o 6.
Sorry for my bad english.

**Describe the bug**

System.DllNotFoundException: "Unable to load shared library 'System.Native' or one of its dependencies. In order to help diagnose loading problems, consider setting the DYLD_PRINT_LIBRARIES environment variable: dlopen(libSystem.Native, 1): image not found"

**Comment by sfratt** [comment]
@anthonystanton I also tried uninstalling the .NET 5.0.6 SDK and dotnet-ef tool of the same version with no luck.

@iezed did uninstalling the 3.1.417 SDK resolve the issue for you?

**Comment by iezed** [comment]
Yes, it's work! I removed all SDKs and reinstalled only 6.0.201 SDK. I'm going to do more tests.

**Comment by anthonystanton** [comment]
Tried uninstalling all versions of SDK and installed SDK 5.0.302 and still having the same issue. 

@sfratt Have you had any luck resolving this?

**Comment by sfratt** [comment]
@anthonystanton I uninstalled all .NET SDKs on my machine and kept just the latest .NET SDK 5.0.406 + .NET 5.0.15 runtime. Reinstalled dotnet-ef 5.0.15 and it worked again for me.

**Comment by ajcvickers** [comment]
Moving to the SDK repo since this does not seem to be an EF-specific bug. but rather an issue with the SDK installed.
